web/src/message_live_update.js

```diff
@@ -66,23 +66,23 @@ export function update_starred_view(message_id, new_value) {
 }
 
 export function update_stream_name(stream_id, new_name) {
-    message_store.update_property("stream_name", new_name, {stream_id});
+    message_store.update_stream_name(new_name, stream_id);
```

Comment:
It’s weird for these parameters to be backwards. (Same with the other three functions.)

Reply:
Yeah fair, I was trying to keep it similar to before, but agree it's weird. I just swapped them to make more sense